core/sail/memory/src/test/java/org/eclipse/rdf4j/sail/memory/benchmark/MemoryBenchmark.java

```diff
@@ -45,6 +49,16 @@
 
 	private List<Statement> statementList = getStatements();
 
+	public static void main(String[] args) throws RunnerException {
+		Options opt = new OptionsBuilder()
+				.include("MemoryBenchmark.load") // adapt to run other benchmark tests
+				// .addProfiler("stack", "lines=20;period=1;top=20")
```

Comment:
Btw. The stack profiler is terrible. It's more often than not gives the wrong results because it only benchmarks safe points. 20 stack lines is also a bit little. I think you need a hundred or maybe more. 

I just Java Flight Recorder for profiling.

Reply:
I just copy pasted that from somewhere but good to know.